File: 프로그래머스/Level1/3진법 뒤집기.cpp
#include <stack>
#include <cmath>

using namespace std;

int solution(int n) {
    int answer = 0;
    stack<int> tmp;
    
    while(n/3 != 0){
        tmp.push(n%3);
        n/=3;
    }
    tmp.push(n);
    
    int i=0;
    while(!tmp.empty()){
        answer += tmp.top()*pow(3,i);
        tmp.pop();
        i++;
    }
        
    return answer;
}

File: 프로그래머스/Level1/3진법 뒤집기.js
function solution(n) {
    let answer = 0;
    let tmp = n.toString(3);

    for(let i=0; i<tmp.length; i++){
        if(tmp[i] === 0) continue;
        answer += tmp[i]*Math.pow(3, i);
    }
    
    return answer;
}
